src/wrappers/themis/wasm/test/test.js

```diff
@@ -77,4 +77,214 @@ describe('wasm-themis', function() {
             })
         })
     })
+    describe('SecureCell', function() {
+        let masterKey1 = new Uint8Array([1, 2, 3, 4])
+        let masterKey2 = new Uint8Array([5, 6, 7, 8, 9])
+        let emptyArray = new Uint8Array()
+        let testInput = new Uint8Array([1, 1, 2, 3, 5, 8, 13])
+        let testContext = new Uint8Array([42])
+        describe('Seal mode', function() {
+            it('does not accept strings', function() {
+                // It's a really nice idea to accept strings as 'master keys', but we have
+                // to define their interpretetaion. So treat them as errors for now.
```

Comment:
decode strings as utf8?

Reply:
Yeah, that's what I was thinking about. Should we settle for UTF-8 across all language wrappers?

IIRC, currently only Java/Android one supports strings as master keys, but it uses UTF-16 encoding. Other wrappers may _accept_ strings, but they will be using some implementation-defined encodings.

Reply:
> Yeah, that's what I was thinking about. Should we settle for UTF-8 across all language wrappers?
I think we shouldn't and you answered why in the next section) Some OS and languages have different default encodings from unicode (windows api use utf-16 too as I remember). So better to support one common format: raw byte arrays. And as addition add some helpers which accept language-native types which are usual for end users